FCCee/ILD_FCCee/compact/ILD_FCCee_v01/FCCdefs_common.xml

```diff
@@ -0,0 +1,103 @@
+  <constant name="Vertex_LumiCal_Clearence" value="110*mrad"/>
+  <!-- END vertex info copied from CLD_o2_v07.xml -->
+
+
+  <!-- _prime marks the coordinate system along the beamline instead of the detector coordinate frame -->
+  <constant name="LumiCal_min_z_prime" value="1074*mm"/>
+  <constant name="LumiCal_dz_prime" value="0.5 * 25 * 4.5 * mm"/> <!-- Must be consistent with layers defined in the lumiCal xml, half-length-->
+  <constant name="LumiCal_max_z_prime" value="LumiCal_min_z_prime + LumiCal_dz_prime * 2" />
+
+  <!-- calculate the Z position in the detector reference frame for consistency with existing drivers -->
+  <constant name="LumiCal_min_z" value="LumiCal_min_z_prime*cos(0.5*CrossingAngle)"/>
+  <constant name="LumiCal_max_z" value="LumiCal_max_z_prime*cos(0.5*CrossingAngle)" />
+
+  <!-- This has nothing to do with the actual length of the LumiCal, the value is only used for the positioning -->
+  <constant name="LumiCal_dz" value="LumiCal_dz_prime * cos(0.5 * CrossingAngle)"/>
+
+  <constant name="LumiCal_inner_radius" value="55.0*mm"/>
+  <constant name="LumiCal_outer_radius" value="115.0*mm- env_safety"/>
+
+  <constant name="LumiCal_Instr_thickness" value="20*mm"/>
+  <constant name="LumiCal_Instr_inner_radius" value="LumiCal_outer_radius"/>
+  <constant name="LumiCal_Instr_outer_radius" value="LumiCal_outer_radius+LumiCal_Instr_thickness - env_safety"/>
+
+  <constant name="LumiCal_Cool_thickness" value="9.75*mm"/>
+  <constant name="LumiCal_Cool_inner_radius" value="LumiCal_Instr_outer_radius"/>
+  <constant name="LumiCal_Cool_outer_radius" value="LumiCal_Instr_outer_radius+LumiCal_Cool_thickness"/>
+
+  <constant name="Lcal_services_rmax" value="LumiCal_outer_radius+30*mm"/>
+  <constant name="Lcal_offset_phi" value=" 0."/>
+  <!--preliminary LumiCal shielding-->
+  <!--back shielding-->
+  <constant name="LumiCal_Shield_inner_radius" value="LumiCal_inner_radius"/>
+  <constant name="LumiCal_Shield_outer_radius" value="LumiCal_outer_radius+LumiCal_Instr_thickness+LumiCal_Cool_thickness"/>
+  <!-- For convenience, z here refers to dimensions *along the beam pipe*, which is tilted by CrossingAngle/2 w.r.t the global z axis -->
+  <constant name="LumiCal_shield_dz_prime" value="3.5/2.0*mm"/> <!-- Must be consistent with layers defined in the lumiCal xml (1*3.5 mm)-->
+  <constant name="LumiCal_shield_dz" value="LumiCal_shield_dz_prime  * cos(0.5 * CrossingAngle)"/>
+
+
+  <constant name="QD0_min_z" value="2000*mm"/>
+  <constant name="QD0_max_z" value="5400*mm"/>
+  <constant name="QD0Coil_outer_radius" value="30*mm"/>
+  <constant name="CollimatorInFrontOfQD0_dz" value="20*cm"/>
+  <constant name="CollimatorInFrontOfQD0_radius" value="10*mm"/>
+  <constant name="CollimatorInFrontOfQD0_dr" value="16*mm"/>
+
+
+
+  <!-- END info copied from CLD_o2_v07.xml -->  
+  <constant name="GlobalTrackerReadoutID" type="string" value="system:5,side:-2,layer:9,module:8,sensor:8"/>
```

Comment:
hmm, not sure about this, to be checked.

Reply:
for other ILD subdetectors, it is hardcoded into the separate xml files for each system.
For tracking subdetectors it is always `system:5,side:2,layer:9,module:8,sensor:8` .

this "GlobalTrackerReadoutID" variable is used for the tracker elements taken from CLD (ie inner tracker, vertex detector). It seems that I adjusted it to (almost) match the ILD standard, but with `side:-2` (I think this was maybe the previous Mokka-time definition...I'm not sure why I did this...)
`system:5,side:-2,layer:6,module:11,sensor:8`
I don't see any reason not to change -2 -> 2 here.

Should we worry that 8 bits may not be enough for "module"?

On the other hand, I believe that @jfklama got track digi, conformal track reconstruction working with the v09 model, which used a mix of encodings, with for the outer tracker
`<id>system:5,side:-2,layer:6,module:11,sensor:8</id>`
Does conformal tracking use the cellID?

[TrackingDigi_SiILD.xml](https://github.com/iLCSoft/ILDConfig/blob/master/StandardConfig/production/Tracking/TrackingDigi_SiILD.xml)
[ConformalTracking_SiILD.xml](https://github.com/iLCSoft/ILDConfig/blob/master/StandardConfig/production/Tracking/ConformalTracking_SiILD.xml)
[HighLevelReco_SiILD.xml](https://github.com/iLCSoft/ILDConfig/blob/master/StandardConfig/production/HighLevelReco/HighLevelReco_SiILD.xml)

I'll look further next week.

Reply:
adjusted to 
` <constant name="GlobalTrackerReadoutID" type="string" value="system:5,side:2,layer:9,module:8,sensor:8"/>`
same as other ILD trackers

Reply:
I think the `8` module bits should be fine for us since there are less modules in the TPC region. I think that is the region where the number of modules will get larger for silicon detectors, such that they require 11 bits.

It's possible that conformal tracking can deal with different cell ID encodings better than the `FullLDCTracking` that is used for ILD.

Reply:
@tmadlener , @Victor-Schwan has got the conformal tracking working (at least to 1st order) with the silicon hits, and Clupatra on the TPC hits; linking the two collections is still to be done. So I think there is no fundamental problems caused by our cellID encodings.

Reply:
Nice :)